packages/salmon-lambda-functions/src/generic/slack.ts

```diff
@@ -1,12 +1,21 @@
+import { Bech32 } from '@defichain/jellyfish-crypto'
 import { OraclesManager } from '@defichain/salmon-oracles-functions'
+import { WhaleApiClient } from '@defichain/whale-api-client'
+import BigNumber from 'bignumber.js'
 import fetch from 'node-fetch'
 import { EnvironmentConfig } from './environment'
 
 export async function checkBalanceAndNotify (oraclesManager: OraclesManager, env: EnvironmentConfig): Promise<void> {
-  const balance = await oraclesManager.getBalance(env.oceanUrl, env.network)
-  if (balance.lt(2.0)) {
-    const address = await oraclesManager.getAddress()
+  const pubKey = await oraclesManager.walletAccount.publicKey()
+  const address = Bech32.fromPubKey(pubKey, 'df')
+
+  const whaleClient = new WhaleApiClient({
+    url: env.oceanUrl,
+    network: env.network
+  })
 
+  const balance = new BigNumber(await whaleClient.address.getBalance(address))
+  if (balance.lt(2.0)) {
```

Comment:
```suggestion
  const balance = await whaleClient.address.getBalance(address)
  if (new BigNumber(balance).lt(2.0)) {
```

try not to nest await in brackets hard to read